plugins/dns_service/app/controllers/dns_service/zones_controller.rb

```diff
@@ -74,7 +75,8 @@ def show
         if not params[:search].blank?
           @search = params[:search]
           @searchfor = "#{params[:searchfor]}"
-          filter = { @searchfor.downcase() => @search }
+          # surrounding the user submitted search term with wildcards is stupid but the easiest way to make the search intuitive and match partial names
+          filter = { @searchfor.downcase() => "*" + @search + "*" }
```

Comment:
same as above, it is better to use `"*#{@search}*"` instead of `"*" + @search + "*"`